fix warnings

BillWagner opened the pull request:
PR #20889 introduced these warnings. This fixes them.

See https://github.com/dotnet/docs/pull/20889#issuecomment-704417583

BillWagner committed:
fix warnings

PR #20889 introduced these warnings. This fixes them.

gewarren reviewed (approved):
And the build is clean. Thanks Bill!

BillWagner committed:
fix build warnings

While it says "passed", the previous commit has warnings to link to removed content.

BillWagner committed:
typo

Youssef1313 commented:
The build still have some warnings.

BillWagner committed:
fix anchors

BillWagner committed:
another typo
